ambari-web/app/mappers/service_metrics_mapper.js

```diff
@@ -192,6 +192,7 @@ App.serviceMetricsMapper = App.QuickDataMapper.create({
     work_status: 'HostRoles.state',
     passive_state: 'HostRoles.maintenance_state',
     display_name: 'HostRoles.display_name',
+    comp_id: 'HostRoles.id',
```

Comment:
Please name this component_id so it follows the pattern.

Reply:
The camel case auto conversion would convert this to compId which is the name of this attribute defined for this model